pkgs/development/libraries/v8/6_x.nix

```diff
@@ -11,6 +11,9 @@ let
          else if stdenv.is64bit
               then"x64"
               else "ia32";
+
+  librarySuffix = if stdenv.isDarwin then "dylib" else "so";
```

Comment:
@Ericson2314 was kind enough to put this knowledge into the stdenv for us! Try `stdenv.hostPlatform.extensions.sharedLibrary`. Note that that one gives you the period as well.

Reply:
Ahh, thanks for pointing this out :) wasn't yet aware